### ISSUE TITLE
Jack-in command launches clojure CLI with no aliases

### ISSUE THREAD

**Issue by taylorwood** [BetterThanTomorrow/calva#234]
If I define a basic deps.edn file with no aliases, the jack-in command still sets the `-A` flag on `clojure` CLI (which is invalid without also passing alias names):
```
> Executing task: /usr/local/bin/clojure -Sdeps "{:deps {nrepl {:mvn/version \"0.6.0\"} cider/cider-nrepl {:mvn/version \"0.21.1\"}}}" -A -m nrepl.cmdline --middleware "[cider.nrepl/cider-middleware]" <

Missing required argument for "-A ALIASES"
The terminal process terminated with exit code: 1
```

Using latest release on VS Code macOS 10.14.

Workaround: add an alias to deps.edn and selecting it when jacking in.

**Comment by PEZ** [comment]
Thanks for reporting. I'm sorry, I think I introduced this with the release of 2.0.18 today...

About the workaround, I don't think you need to select the dummy alias.

I'll fix this in a few hours. Just need a computer!

**Comment by PEZ** [comment]
Yes, it was indeed something I had introduced today... Lucky us that you spotted it and reported!

I've just published `2.0.19` which fixes this. (Might take a few minutes before it appears on the Marketplace.) Please close this issue if the fix checks out for you, @taylorwood .

**Comment by taylorwood** [comment]
Fix confirmed, thanks!
